### ISSUE TITLE
[Suggestion]: Wither Storm gets distracted by nearby structures instead of farther ones.

### ISSUE THREAD

**Issue by TaintedBlock** [nonamecrackers2/crackers-wither-storm-mod#1835]
### Description of feature request

The Wither Storm likes to go for structures that are farther away when getting distracted instead of the ones right infront of it, which I think doesn't really make too much sense.

**Comment by nonamecrackers2** [comment]
Does it? I thought it picks the nearest one

**Comment by Aimyx** [comment]
Wither storm got adhd.
Can’t focus on the house right next to it

**Comment by sm644me** [comment]
This is something I ran into while developing Explorer's Eve, not even related to the Wither Storm. It looks like if you have multiple structures in a tag, /locate behaves oddly, and CWSM seems to use a function very similar to the in-game /locate. I'm not sure if this is something that'd be practical for Crackers to fix, though it is weird and sometimes annoying.

**Comment by TaintedBlockmObile** [comment]
> Does it? I thought it picks the nearest one

Yeah, I actually saw it happen for myself. I even made the wither storm distracted right next to a desert temple and it instead went to something like 300 blocks away from it if not further

**Comment by nonamecrackers2** [comment]
> This is something I ran into while developing Explorer's Eve, not even related to the Wither Storm. It looks like if you have multiple structures in a tag, /locate behaves oddly, and CWSM seems to use a function very similar to the in-game /locate. I'm not sure if this is something that'd be practical for Crackers to fix, though it is weird and sometimes annoying.

Yeah it uses whatever /locate uses. I assume since it’s a tag with a bunch of structures, it might first try to find the first nearby structure at the beginning of the tag, which might not always be the closest structure it’s allowed to target? I’m not sure to be honest
